From b4442fb408c80cf6c4d8fd05b474c0094e243126 Mon Sep 17 00:00:00 2001
Message-Id: <b4442fb408c80cf6c4d8fd05b474c0094e243126.1573779465.git.liu.denton@gmail.com>
In-Reply-To: <cover.1573779465.git.liu.denton@gmail.com>
References: <cover.1573779465.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Mon, 28 Oct 2019 22:39:10 -0700
Subject: [PATCH 01/27] lib-bash.sh: move `then` onto its own line
To: Git Mailing List <git@vger.kernel.org>

The code style for tests is to have statements on their own line if
possible. Move the `then` onto its own line so that it conforms with the
test style.

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
 t/lib-bash.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/t/lib-bash.sh b/t/lib-bash.sh
index 2be955fafb..6a2c21cd24 100644
--- a/t/lib-bash.sh
+++ b/t/lib-bash.sh
@@ -2,7 +2,8 @@
 # to run under Bash; primarily intended for tests of the completion
 # script.
 
-if test -n "$BASH" && test -z "$POSIXLY_CORRECT"; then
+if test -n "$BASH" && test -z "$POSIXLY_CORRECT"
+then
 	# we are in full-on bash mode
 	true
 elif type bash >/dev/null 2>&1; then
-- 
2.24.0.399.gf8350c9437

